docker/hdfs/files/build-hadoop.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 
-test -z "${HADOOP_VERSION}" && HADOOP_VERSION=3.2.2
+test -z "${HADOOP_VERSION}" && HADOOP_VERSION=3.3.3
```

Comment:
Have you tested this script after bumping the Hadoop version?

Reply:
It does work, but this is a good question I can't remember why this was changed, I think in line with the Gaffer [change](https://github.com/gchq/Gaffer/blob/549a612c11d6d911c63cd353b2a4b9e243a72275/pom.xml#L68), but I suppose they don't have to align.